.gitignore

```diff
@@ -1,3 +1,5 @@
+ProfCalculator/.classpath
+.project
```

Comment:
Probably not a got idea to ignore these files. These are needed for others to import the project.